src/main/webapp/app/entities/result/submission-result-status.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="col-auto" [ngSwitch]="exercise.participationStatus">
+    <!-- On desktop -->
+    <div class="d-none d-sm-inline">
+        <span class="text-muted" *ngSwitchCase="'uninitialized'">{{ 'artemisApp.courseOverview.exerciseList.userNotStartedExercise' | translate }}</span>
```

Comment:
I guess here you need to cover both on desktop and mobile the same strings. Because right now mobile misses `exercise-missed` and `exercise-active` keys.

Reply:
Good catch. This is also the case on production right now, so we can fix this as part of this refactoring. I added the two missing cases for mobile.